src/wandbot/rag/query_handler.py

```diff
@@ -279,8 +279,8 @@ def chain(self) -> Runnable:
         return self._chain
 
     def _load_chain(self, model: ChatOpenAI) -> Runnable:
-        query_enhancer_chain = create_structured_output_runnable(
-            EnhancedQuery, model, self.prompt
+        query_enhancer_chain = self.prompt | model.with_structured_output(
```

Comment:
again this is new LECL api?

Reply:
Nope, just convenience to use `weave.op` decorator